src/core/Akka.Streams.TestKit.Tests/Akka.Streams.TestKit.Tests.csproj

```diff
@@ -41,8 +41,8 @@
       <HintPath>..\..\packages\FluentAssertions.4.2.1\lib\net45\FluentAssertions.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Reactive.Streams, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Reactive.Streams.1.0.0\lib\net40\Reactive.Streams.dll</HintPath>
+    <Reference Include="Reactive.Streams, Version=1.0.1.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Reactive.Streams.1.0.1\lib\net45\Reactive.Streams.dll</HintPath>
```

Comment:
Good catch, I'll look into it. Most likely because I am now using fake to build the `Reactive.Streams` package, I probably need to change the script to target 4.0.